clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/DisableServiceAtomicOperation.java

```diff
@@ -52,16 +54,41 @@ private static Task getTask() {
 
   @Override
   public Void operate(List priorOutputs) {
-    getTask().updateStatus(BASE_PHASE, "Initializing Disable Amazon ECS Server Group Operation...");
+    updateTaskStatus("Initializing Disable Amazon ECS Server Group Operation...");
+    disableService();
+    return null;
+  }
 
-    AmazonCredentials credentials = (AmazonCredentials) accountCredentialsProvider.getCredentials(description.getCredentialAccount());
-    AmazonECS ecs = amazonClientProvider.getAmazonEcs(description.getCredentialAccount(), credentials.getCredentialsProvider(), description.getRegion());
+  private void disableService() {
```

Comment:
Please put arguments to this method, and perhaps break it down into a few methods

Reply:
all of the information required for this method to work is contained in the DisableServiceAtomicOperation instance. It would be suboptimal to extract that logic into the operate method and then pass a bunch of arguments to the disableService. 

When we have a testing strategy ironed out we can refactor for testability. 

Broke off getting the cluster value into its own method, since the region variable is not actually used in the call